src/fsc/fscProject/fsc.fsproj

```diff
@@ -10,15 +10,8 @@
 
   <PropertyGroup Condition="'$(Configuration)' == 'Proto'">
     <TargetFramework>$(FSharpNetCoreProductTargetFramework)</TargetFramework>
-    <PublishReadyToRun>$(EnablePublishReadyToRun)</PublishReadyToRun>
-    <RuntimeIdentifier>$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(Configuration)' == 'Proto'">
-    <PublishReadyToRunExclude Include="FSharp.Build.dll" />
-    <PublishReadyToRunExclude Include="FSharp.Core.dll" />
-  </ItemGroup>
-
```

Comment:
Why was it removed? Publishing r2r proto was the point to get rid of full framework proto, to make it reasonable fast.

Reply:
It did not work with FSharp.Build.
For a bootstrapped FSharp.Build. it cannot be R2R - I did not actually attempt to modify the build tasks to make them R2R friendly, as I only see this as an ephemeral need.

Once nullness is in LKG, bootstrapped Fsharp.Build can be reverted.

Reply:
@vzarytovskii , @KevinRansom  :

As discussed, I will:
- Repeat with keeping the `PublishReadyToRunExclude` exclusions, and report the exact issues (dotnet build, as well as msbuild proto process)
- If this needs to be in until nullnes lands in LKG, condition the changes in a $(BootstrapFsharpBuild) property